tests/data/conf_covariance_cNG_fsky.yaml

```diff
@@ -0,0 +1,68 @@
+tjpcov:
+  # sacc input file 
+  sacc_file: ./tests/benchmarks/32_DES_tjpcov_bm/cls_cov.fits
+
+  # 'set' from parameters OR pass CCL cosmology object OR yaml file
+  cosmo: 'set'
+
+  # Setting mask OR fsky approximation
+  mask_file:
+    DESgc__0: ./tests/benchmarks/32_DES_tjpcov_bm/catalogs/mask_DESgc__0.fits.gz
+    DESwl__0: ./tests/benchmarks/32_DES_tjpcov_bm/catalogs/DESwlMETACAL_mask_zbin0_ns32.fits.gz
+    DESwl__1: ./tests/benchmarks/32_DES_tjpcov_bm/catalogs/DESwlMETACAL_mask_zbin1_ns32.fits.gz
+
+  mask_names:
+    DESgc__0: mask_DESgc0
+    DESwl__0: mask_DESwl0
+    DESwl__1: mask_DESwl1
+
+  outdir: ./tests/tmp/
+
+  # Survey params:
+  # 5 lens bins
+  Ngal_DESgc__0: 26
+
+  Ngal_DESwl__0: 26
+  Ngal_DESwl__1: 26
+  # # constant bin sigma_e
+  sigma_e_DESwl__0: 0.26
+  sigma_e_DESwl__1: 0.26
+
+  # linear bias for lenses constant for redshift bin (example notebook)
+  bias_DESgc__0: 1.48
+
+  # IA: 0.5
+
+parameters:
+  # Not used for while (read by ccl.cosmo):
+  Omega_c: 0.2640
+  Omega_b: 0.0493
+  h: 0.6736
+  n_s: 0.9649
+  sigma8: 0.8111
+  w0: -1
+  wa: 0
+  transfer_function: 'boltzmann_camb'
+HOD:
+  # automatically creates massdef and concentration objects 
+  log10Mmin_0: 12.0
+  log10Mmin_p: 0.0
+  siglnM_0: 0.4
+  siglnM_p: 0.0
+  log10M0_0: 7.0
+  log10M0_p: 0.0
+  log10M1_0: 13.3
+  log10M1_p: 0.0
+  alpha_0: 1.0
+  alpha_p: 0.0
+  fc_0: 1.0
+  fc_p: 0.0
+  bg_0: 1.0
+  bg_p: 0.0
+  bmax_0: 1.0
+  bmax_p: 0.0
+  a_pivot: 1.0
+  ns_independent: False
+
+GaussianFsky:
```

Comment:
Since now more components apart from the Gaussian part will be using fsky, could you move this to the `tjpcov` section? i.e.

```
tjpcov:
    fsky: 0.05
```

(and update the other parts of the code correspondingly)